doc/source/feature_comparison.rst

```diff
@@ -20,111 +20,260 @@
-|Improved Handling of Corrupted Tables                   | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|Ability to Kill Idle Transactions                       | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-|START TRANSACTION WITH CONSISTENT SNAPSHOT improvements | Yes                  |              |
-+--------------------------------------------------------+----------------------+--------------+
-
-
-+---------------------------------------+----------------------+--------------+
-|Running Database as a Service          |Percona Server 5.7.10 |MySQL 5.7.10  |
-+=======================================+======================+==============+
-|Special Utility User                   | Yes                  |              |
-+---------------------------------------+----------------------+--------------+
-|Expanded Program Option Modifiers      | Yes                  |              |
-+---------------------------------------+----------------------+--------------+
-|Enforcing the Specific Storage Engine  | Yes                  |              |
-+---------------------------------------+----------------------+--------------+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+    :header-rows: 1
+
+    * - Features
+      - Percona Server 5.7.24
+      - MySQL 5.7.24
+    * - Open source
+      - Yes
+      - Yes
+    * - ACID Compliance
+      - Yes
+      - Yes
+    * - Multi-Version Concurrency Control
+      - Yes
+      - Yes
+    * - Row-Level Locking
+      - Yes
+      - Yes
+    * - Automatic Crash Recovery
+      - Yes
+      - Yes
+    * - Table Partitioning
+      - Yes
+      - Yes
+    * - Views
+      - Yes
+      - Yes
+    * - Subqueries
+      - Yes
+      - Yes
+    * - Triggers
+      - Yes
+      - Yes
+    * - Stored Procedures
+      - Yes
+      - Yes
+    * - Foreign Keys
+      - Yes
+      - Yes
+    * - GTID Replication
+      - Yes
+      - Yes
+    * - Group Replication
+      - Yes
+      - Yes
+    * - MyRocks Storage Engine
+      - Yes
+      - No
+    * - TokuDB Storage Engine
+      - Yes
+      - No
+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+   :header-rows: 1
+   
+   * - Extra Features for Developers
+     - Percona Server 5.7.24
+     - MySQL 5.7.24
+   * - NoSQL Socket-Level Interface
+     - Yes
+     - Yes
+   * - X API Support
+     - Yes
+     - Yes
+   * - InnoDB Full-Text Search Improvements
+     - Yes
+     - No
+   * - Extra Hash/Digest Functions
+     - Yes
+     - No
+
+.. tabularcolumns:: |p{5cm}|p{5cm}|p{5cm}|
+
+.. list-table::
+   :header-rows: 1
+
+   * - Instrumentation and Troubleshooting Features
+     - Percona Server 5.7.24
+     - MySQL 5.7.24
+   * - INFORMATION_SCHEMA Tables
+     - 93
```

Comment:
This was 75 in the original table - did we implement 18 new since then?

Reply:
According to the PS Feature Comparison spreadsheet made during 8.0 GA (Tyler)
INFORMATION_SCHEMA Tables - 93

Reply:
I've just executed `show tables` on the current 5.7 turnk, and I see 71 tables - not even 75. So it would be great for somebody who actually does release testing of our software (QA?) to verify these numbers.

Reply:
Yes, we should have some doc QA, but realistically, this to me seems like 100% useless information anyway.  x number of  i_s tables vs y makes not one bit of difference to an end user.  What they do care about is what those tables do.

Reply:
The other number (432  or 789) is more significant, this one is a smaller difference, but still. I would go with the smaller numbers then - either the original, or what I see with running the server.

Reply:
changed to 71.